iphone/Classes/TiUIWebView.m

```diff
@@ -488,6 +488,13 @@ -(void)setScrollsToTop_:(id)value
 	[[self scrollview] setScrollsToTop:scrollsToTop];
 }
 
+- (void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE(args, NSString);
+    NSDictionary *dict = [NSDictionary dictionaryWithObjectsAndKeys:[TiUtils stringValue:args], @"UserAgent", nil];
+    [[NSUserDefaults standardUserDefaults] registerDefaults:dict];
```

Comment:
The user must be able to reset it by passing `null` to get back the old native behavior. That's what prevented us from doing it before.